wordpress-mobile/WordPress-Android
# Fixing material theme on older API's and removing inconsistencies

This PR fixes drawable tinting on older devices (API 21-23) and removes some minor inconsistencies with styling around the app.

What changed:
- Bumped appcompat to `1.2.0-alpha03`. This was necessary because of the nasty bug in appcompat 1.1.0 (yes, another bug) that crashed WebView.
- Updated tinting logic to be older-api friendly:
    - Using `app:tint` instead of `android:tint` (this is 85% of changes)
    - Using alpha values inside selectors instead of using nested selectors with alpha inside them.
    - Using `AppCompatResources` to get `ColorStateList`
- Changed `accentColor` from secondary to primary.
- Normalized icon color through the app. Now, most icons should use only one shade of gray and not 50.
- Global style value for material cards - since we are using Bridge version of the material theme, it's not automatically picked up by cards, and without style, they are too elevated.

To test:
- Launch the app on device with API 21 and do a brief look around. Apart from icon color, card elevation and accent color you should see no changes in appearance. 

If you see any icons that look like this, let me know:
[![Image from Gyazo](https://i.gyazo.com/8f8c125e3f2c0b21feeb2109604e834b.png)](https://gyazo.com/8f8c125e3f2c0b21feeb2109604e834b)

- Launch the app on modern devices with API 27, 28 or 29. and do a brief look around. Apart from icon color, card elevation and accent color you should see no changes in appearance. 

- On any device, navigate to site creation flow, and make sure that the domain picker field works as expected.

PR submission checklist:
